src/components/Course.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import '../public/index.css'
```

Comment:
I think our style guide asks for double quotes here (and on other imports as well). 

If we want to make an exception for import that is fine with me, but will need to make a note of updating the guide/Lint.

Reply:
The vast majority of import examples I'm seeing online use single quotes, _however_, I don't think we should make an exception. In my opinion, we should either stick to our standard or change it instead of cutting out edge cases. If we _wanted_ to switch to single quotes, template strings would still give us a way to use both types of quotes. Food for thought.

Reply:
I'm going with the double-quotes. The only exceptions I'd change the quoting are template literals and to avoid escape characters.